package.json

```diff
@@ -11,8 +11,10 @@
     "url": "https://github.com/JedWatson/react-select.git"
   },
   "dependencies": {
+    "create-react-class": "15.5.1",
     "classnames": "^2.2.4",
-    "react-input-autosize": "^1.1.0"
+    "react-input-autosize": "^1.1.0",
```

Comment:
The lib has been updated, fixed version is [1.1.1](https://github.com/JedWatson/react-input-autosize/pull/79/files/6bb9330dcf473d1252d67988ef2cc069aa297764)